gradle.properties

```diff
@@ -1,3 +1,5 @@
+version=0.0.18-SNAPSHOT
```

Comment:
Har det noe å si (for workflowen?) om denne ligger her eller i build.gradle.kts?

Reply:
Jeg mener å huske at version i gradle.properties gjør det lettere å kunne overstyre fra kommandolinjen dersom man bygger lokalt og vil sette en egen versjon - og at dette ikke funker dersom version ligger i build.gradle - men tror ikke dette egentlig har noe å si for workflowen